samza-core/src/test/java/org/apache/samza/container/TestRunLoop.java

```diff
@@ -231,14 +251,12 @@ public void testProcessMultipleTasks() throws Exception {
     when(consumerMultiplexer.pollIntervalMs()).thenReturn(10);
```

Comment:
Is this unused now? Could you please check the other tests to see if there are other unused variables too?

Reply:
Done